searchlib/src/vespa/searchlib/fef/phrasesplitter.h

```diff
@@ -72,6 +75,7 @@ class PhraseSplitter : public PhraseSplitterQueryEnv
     }
 
     void bind_match_data(const fef::MatchData &md) { _matchData = &md; }
+    const PhraseSplitterQueryEnv& get_phrase_splitter_query_env() const { return _phrase_splitter_query_env; }
```

Comment:
Could consider to reduce the verbosity if this function name. E.g. _get_query_env()_ instead.